pkg/webhook/clusterresourceplacement/clusterresourceplacement_webhook.go

```diff
@@ -16,6 +16,6 @@ const (
 	ValidationPath = "/validate-fleet-azure-com-v1alpha1-clusterresourceplacement"
 )
 
-func Add(mgr manager.Manager) error {
+func Add(mgr manager.Manager, _ []string) error {
```

Comment:
_ []string is used as a placeholder for now, this file will eventually be removed in follow up PRs, same goes for the separate handler files for pods and replicasets